docs/src/main/sphinx/release/release-407.md

```diff
@@ -8,37 +8,96 @@
+* Fix listing relations failure when a Glue table has no table type set. ({issue}`15909`)
 
 ## Hive connector
 
-* Improve performance of reading decimal columns from Parquet files. ({issue}`15713`)
-* Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
 * Reduce query latency. ({issue}`15811`)
+* Improve performance of reading decimal types from Parquet files. ({issue}`15713`)
+* Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
+* Improve performance of reading string types from Parquet files. ({issue}`15897`, {issue}`15923`)
+* Improve performance of reading timestamp and boolean types from Parquet files. ({issue}`15954`)
+* Fix query failure when reading Parquet files written by Impala. ({issue}`15942`)
+* Fix listing relations failure when a Glue table has no table type set. ({issue}`15909`)
 
 ## Hudi connector
 
-* Improve performance of reading decimal columns from Parquet files. ({issue}`15713`)
+* Improve performance of reading decimal types from Parquet files. ({issue}`15713`)
 * Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
+* Improve performance of reading string types from Parquet files. ({issue}`15897`, {issue}`15923`)
+* Improve performance of reading timestamp and boolean types from Parquet files. ({issue}`15954`)
+* Fix query failure when reading Parquet files written by Impala. ({issue}`15942`)
 
 ## Iceberg connector
 
 * Add support for the `unregister_table` procedure. ({issue}`15784`)
+* Add the `iceberg.jdbc-catalog.connection-user` and
+  `iceberg.jdbc-catalog.connection-password` configuration properties to the
+  JDBC catalog. ({issue}`16040`)
+* Add support for compacting manifests asynchronously, which can be enabled by
+  setting the `iceberg.merge_manifests_on_write` session property to `false`. ({issue}`14822`)
+* Improve performance of `DROP TABLE`. ({issue}`15981`)
+* Improve performance of reading [position delete files](https://iceberg.apache.org/spec/#position-delete-files)
+  with ORC data ({issue}`15969`).
 * Improve performance of reading decimal columns from Parquet files. ({issue}`15713`)
 * Improve performance of reading numeric types from Parquet files. ({issue}`15850`)
+* Improve performance of reading string types from Parquet files. ({issue}`15897`, {issue}`15923`)
+* Improve performance of reading timestamp and boolean types from Parquet files. ({issue}`15954`)
+* Prevent creating a table when the specified schema does not exist. ({issue}`15779`)
+* Fix query failure when reading Parquet files written by Impala. ({issue}`15942`)
+* Fix listing relations failure when a Glue table has no table type set. ({issue}`15909`)
+* Fix failure when encountering access denied exceptions while listing
+  materialized views in the Glue metastore. ({issue}`15893`)
+
+## Kudu connector
+
+* Fix authentication failure when Kerberos tickets expire. ({issue}`14372`)
```

Comment:
is it really called a Kerberos ticket ?

Reply:
[Yep!](https://web.mit.edu/kerberos/krb5-latest/doc/user/tkt_mgmt.html)

Reply:
Wow.. crazy